### PR TITLE
Add max-width to right half of hero section

### PR DESCRIPTION
**Pull request by kelseyll**
- [x] Add max-width for right half of hero
- [x] Add character limit on club title/description

Once I can test this in local I will assign someone to review

### CODE REVIEW

File: client/components/pages/editor/sections/Hero.jsx [hunk 1]
```diff
@@ -27,8 +27,8 @@ export default class Hero extends Component{
   render(){
     return (
       <form>
-        <ShortTextField label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
-        <LongTextField label="Description" onChange={this.handleChange} value={this.props.data.description} name="description"/>
+        <ShortTextField maxLength="60"label="Title" onChange={this.handleChange} value={this.props.data.title} name="title"/>
```
Comment (hwkr) [inline client/components/pages/editor/sections/Hero.jsx]: I don't think this code will work. Did you test it?
(These text fields are react components so `maxLength` isn't a property. You should pass it down into the components as props and apply it to the internal `<inputs>`).

**Review by hwkr — CHANGES_REQUESTED** [review]
We should make this work.

File: client/components/pages/editor/sections/Hero.jsx (hunk 1 above)
Comment (kelseyll) [reply client/components/pages/editor/sections/Hero.jsx]: I haven't tested it, hence the "Once I can test this in local I will assign someone to review" in the pr description.

You are probably right, I read this from the react docs:
`In React, all DOM properties and attributes (including event handlers) should be camelCased. For example, the HTML attribute tabindex corresponds to the attribute tabIndex in React.`

https://facebook.github.io/react/docs/dom-elements.html

So next step (once I test it) is to find where the actual inputs are, and then add the `maxLength="__"` to that file? I can do that tonight. 🌃

**Comment by hwkr** [comment]
Thoughts: maybe we want to pass down the max length as a React prop rather than hardcoding it for every field.

**Comment by kelseyll** [comment]
If you think you could do that super quickly then I'm down. Honestly though, I don't think code optimization is really what we need to be focusing on right now (we have 7 days until presentation) 💯

**Review by hwkr — APPROVED** [review]

**Comment by hwkr** [comment]
Okay, I'll just merge it then.

**Pull request merged**